templates/enhancement/enhance_file.py

```diff
@@ -22,8 +23,7 @@
     args = parser.parse_args()
 
     enhancer = SpectralMaskEnhancement.from_hparams(
-        source=".",
-        hparams_file="inference.yaml",
-        savedir=args.save_directory,
+        source=os.path.abspath(args.save_directory),
+        hparams_file=os.path.join(os.getcwd(), "inference.yaml"),
```

Comment:
This doesn't work because the hparams path is appended to the source path. Although maybe it should detect absolute paths and stop trying to concatenate them...